vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -410,12 +412,16 @@ For more details on Content Type properties, refer to the [Content Type document
 ## Content
 This descriptor enables content to be created and optionally published via a bundle, according to the `status` property. The content ID is optional and enables linking from other components, like Content Widget. It can be autogenerated or explicitly declared.
 
-**Content descriptor.yaml.**
+Groups in a content descriptor are configured by the `mainGroup` and `groups` properties. A group designation of "free" signifies a public entity.
```

Comment:
'...by the `mainGroup` and `groups` properties. A group designation of "free" signifies a public entity.' -> 'by the `mainGroup` and `groups`, for the owner and join groups, respectively.  The `mainGroup` is for the users who can manage the content within the AppBuilder and `groups` for those users who can view the content.`